sycl/source/device_selector.cpp

```diff
@@ -188,6 +187,33 @@ int host_selector::operator()(const device &dev) const {
   return Score;
 }
 
+int aspect_selector_t::operator()(const device &Dev) const {
+  // neither aspect from deny list is allowed
```

Comment:
instead of two faceless for loops, perhaps it would be clearer to use std::find and std::all_of ?

Reply:
Done.